setup.py

```diff
@@ -21,7 +21,8 @@
               'bluesky.callbacks'],
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'prettytable', 'cycler',
-                      'numpy', 'matplotlib', 'super_state_machine'],
+                      'numpy', 'matplotlib', 'super_state_machine',
+                      'historydict'],
```

Comment:
Does conda build for this recipe work? I remember getting errors in conda build about `downloading with setuptools is disabled during conda build` but I never tracked down exactly what caused it. I know that removing a bunch of stuff from `install_requires` made the error go away. That was the source of my initial comment